docs/commands/super.md

```diff
@@ -139,14 +143,14 @@ The input format is typically [detected automatically](#auto-detection) and the
 "Auto" is "yes" in the table above support _auto-detection_.
 Formats without auto-detection require the `-i` option.
 
-### Hard-wired Input Format
+#### Hard-wired Input Format
```

Comment:
In #5481, a new "Data Formats" section was created and the "Input Formats" and "Output Formats" sub-sections were moved below that, which all makes sense. However, the sub-sections under "Input Formats" and "Output Formats" were left at their prior level, so the hierarchical order was disrupted. This and other `####`-style changes in this PR are to correct that.